manifests/server/options.pp

```diff
@@ -22,6 +22,7 @@
   $check_names_slave = undef,
   $check_names_response = undef,
   $allow_query = [],
+  $no_dns_sec = false,
```

Comment:
Can you also add a docstring for this parameter? The style so far is not great, can you start conforming to the styleguide with something like:
```
# == Paramaters
#
# [*dnssec_validation*]
# Bool, defaults to true. If false disabled dnssec and dnssec-validation.
#